src/goto-instrument/contracts/havoc_assigns_clause_targets.h

```diff
@@ -0,0 +1,62 @@
+/*******************************************************************\
+
+Module: Havoc multiple and possibly dependent targets simultaneously
+
+Author: Remi Delmas, delmasrd@amazon.com
+
+\*******************************************************************/
+
+/// \file
+/// Havoc function assigns clauses
+
+#ifndef CPROVER_GOTO_INSTRUMENT_CONTRACTS_HAVOC_ASSIGNS_CLAUSE_TARGETS_H
+#define CPROVER_GOTO_INSTRUMENT_CONTRACTS_HAVOC_ASSIGNS_CLAUSE_TARGETS_H
+
+#include <set>
+
+#include <goto-programs/goto_program.h>
```

Comment:
I think you'll only need `#include <util/expr.h>`, everything else can be forward-declared.

Reply:
indeed this works